### ISSUE TITLE
Random Training Stalls on audio training

### ISSUE THREAD

**Issue by rongkunxue** [youngsheen/SimVQ#9]
![Snipaste_2024-12-06_14-47-09](https://github.com/user-attachments/assets/8b72c7f8-59bb-4588-a633-25c4e7845bc1)
![Snipaste_2024-12-06_14-51-49](https://github.com/user-attachments/assets/9b1255ab-ca1e-4718-bc78-a52dc532f42a)
Hello, I frequently encounter the situation shown in the image below during training using the config from the library (without making any modifications). In this scenario, the GPU status is as shown in the second image. However, the training loss appears to be normal. I have observed that this issue might occur at the 5th epoch, the 40th epoch, the 49th epoch, or it may not appear at all. Have you encountered this issue before?

**Comment by youngsheen** [comment]
Hi, I haven’t encountered this issue before, and I tend to believe it’s a problem with your server. By the way, have you experienced NaN issues, and how’s the convergence performance of your codebook utilization?

Thanks

**Comment by rongkunxue** [comment]
Thank you for your response. I haven’t encountered any NaN issues. [wandb log](https://wandb.ai/rongkun-china/tmp/runs/xlhjk7mn?nw=nwuserrongkun) is normal, and even after the issue occurred at epoch 46, I was able to restore the normal sound after evaluation. I also tend to believe it’s a problem with my server. Could you tell me the Python version and PyTorch version in your environment? I am using torch 2.5.0, torchaudio 2.5.0, and Python 3.10. Thank you again.

**Comment by youngsheen** [comment]
The environment version is the same to me. I do not think Python version and PyTorch version will result in this issue.

I also noticed that you seem to use a much larger batch size for training, which requires more epochs for the codebook utilization to reach 100%.
